tests/testthat/test-as_polars_series.R

```diff
@@ -79,7 +79,7 @@ test_that("as_polars_series.default throws an error", {
 test_that("as_polars_series.polars_expr throws an error", {
   expect_error(
     as_polars_series(pl$lit(1)),
-    r"(You can evaluating the expression with `pl\$select\(\))"
+    "You can evaluating the expression with `pl\\$select\\(\\)"
```

Comment:
From my digging, the problem here seems to be a mismatch in the number of backticks, and I think we can keep the Raw strings.

Reply:
Done